openbb_platform/extensions/tests/utils/router_testers.py

```diff
@@ -8,13 +8,22 @@
 from openbb_core.app.provider_interface import ProviderInterface
 
 
-def collect_routers(path) -> List:
-    """Collect all routers in the path."""
+def collect_routers(target_dir: str) -> List[str]:
+    """Collect all routers in the target directory."""
+    current_dir = os.path.dirname(__file__)
+    base_path = os.path.abspath(os.path.join(current_dir, "../../../"))
```

Comment:
Is there another way to grab the `base_path`? In case of anything changing locations, this might break.

Reply:
Not the best way, I know. Not sure how to do it right now. But I don't envision the test file moving around the codebase either :D